polkadot/cli/src/command.rs

```diff
@@ -451,7 +451,7 @@ pub fn run() -> Result<()> {
 
 					if cfg!(feature = "runtime-benchmarks") {
 						runner.sync_run(|config| {
-							cmd.run::<service::Block, ()>(config)
+							cmd.run::<sp_runtime::traits::HashingFor<service::Block>, ()>(config)
```

Comment:
Block (the trait I guess) seems to be reexported for service, wonder if it can make sense to also reexport HashingFor and avoid importing sp_runtime to this crate.

Reply:
Yea could do, but i rather want to minimize the number of exports. It does make things more ergonomic, but its also an anti-pattern IMHO.